button.js

```diff
@@ -11,17 +11,15 @@ function replaceWord() {
   fetch("https://wordsapiv1.p.rapidapi.com/words/?random=true", {
     method: "GET",
     headers: {
-      "x-rapidapi-key": "your-api-key-here",
-      "x-rapidapi-host": "api-host-here",
+      "x-rapidapi-key": "keys",
+      "x-rapidapi-host": "wordsapiv1.p.rapidapi.com",
     },
   })
     .then((response) => {
-      console.log(response);
-      response.json().then(() => {
-        /* 
-          Complete this handler function with code that populates an html 
-          element with the random word 
-        */
+      response.json().then((responseBody) => {
+        console.log(responseBody.word);
```

Comment:
We don't need logging statements in finished code.

Reply:
i removed it in the new attempt